mne/forward/tests/test_make_forward.py

```diff
@@ -354,4 +360,107 @@ def test_forward_mixed_source_space():
                   mri_resolution=True, trans=vox_mri_t)
 
 
+@slow_test
+@testing.requires_testing_data
+def test_make_forward_dipole():
+    """Test forward-projecting dipoles"""
+    rng = np.random.RandomState(0)
+
+    evoked = read_evokeds(fname_evo)[0]
+    info = evoked.info
+    cov = read_cov(fname_cov)
+    dip_c = read_dipole(fname_dip)
+
+    # Make new Dipole object with n_test_dipoles picked from the dipoles
+    # in the test dataset.
+    n_test_dipoles = 5
+    dipsel = np.sort(rng.permutation(range(len(dip_c)))[:n_test_dipoles])
```

Comment:
this *might* bomb on Py3k because `range` is an iterable there, not a list. In that case you can use `np.arange(...)` or `list(range(...))`.